docs/desi/desi.rst

```diff
@@ -0,0 +1,111 @@
+    9907734382838564 38.12840803351445 166.09921863682337 ...  EXP      1667p378
+    9907734382838584 38.12836885301038 166.10070750146636 ...  REX      1667p378
+
+The result is an astropy.Table.
+
+Get images
+===============
+
+To download images for a certain region of interest,
+we can define our region in the same way used in the example above.
+
+.. code-block:: python
+
+    >>> from astroquery.desi import DESILegacySurvey
+    >>> from astropy.coordinates import Angle, SkyCoord
+    >>> ra = Angle('11h04m27s', unit='hourangle').degree
+    >>> dec = Angle('+38d12m32s', unit='hourangle').degree
+    >>> pos = SkyCoord(ra, dec, unit='degree')
+    >>> radius = Angle(0.5, unit='arcmin')
+    >>> pixels = 60
+    >>> im = DESILegacySurvey.get_images(pos, data_release=9, radius=radius, pixels=pixels)
+
+All the information we need can be found within the object "im".
+
+.. code-block:: python
+
+    >>> hdul = im[0]
+    >>> hdul[0].header
+
+    SIMPLE  =                    T / file does conform to FITS standard
+    BITPIX  =                  -32 / number of bits per data pixel
+    NAXIS   =                    2 / number of data axes
+    NAXIS1  =                   60 / length of data axis 1
+    NAXIS2  =                   60 / length of data axis 2
+    EXTEND  =                    T / FITS dataset may contain extensions
+    COMMENT   FITS (Flexible Image Transport System) format is defined in 'Astronomy
+    COMMENT   and Astrophysics', volume 376, page 359; bibcode: 2001A&A...376..359H
+    BANDS   = 'g       '
+    BAND0   = 'g       '
+    CTYPE1  = 'RA---TAN'           / TANgent plane
+    CTYPE2  = 'DEC--TAN'           / TANgent plane
+    CRVAL1  =             166.1125 / Reference RA
+    CRVAL2  =     38.2088888888889 / Reference Dec
+    CRPIX1  =                 30.5 / Reference x
+    CRPIX2  =                 30.5 / Reference y
+    CD1_1   = -0.000277777777777778 / CD matrix
+    CD1_2   =                   0. / CD matrix
+    CD2_1   =                   0. / CD matrix
+    CD2_2   = 0.000277777777777778 / CD matrix
+    IMAGEW  =                  60. / Image width
+    IMAGEH  =                  60. / Image height
+
+The variable "im" is a list of `~astropy.io.fits.HDUList` objects, one entry for
+each corresponding object.
+
+In case a set of not valid coordinates is provided, then a `astroquery.exceptions.NoResultsWarning`
+exception is raised, as shown in the example below.
+
+.. code-block:: python
+
+    >>> from astroquery.desi import DESILegacySurvey
+    >>> from astropy.coordinates import Angle, SkyCoord
+    >>> ra = Angle('86.633212', unit='degree').degree
+    >>> dec = Angle('22.01446', unit='degree').degree
+    >>> radius_input = 3
+    >>> pixels = 1296000
+
+    >>> pos = SkyCoord(ra, dec, unit='degree')
+    >>> radius = Angle(radius_input, unit='arcmin')
+    >>> pixels = 60
+    >>> im = DESILegacySurvey.get_images(pos, data_release=9, radius=radius, pixels=pixels)
+
+.. code-block:: python
+
```

Comment:
you need to write the whole example script as one big block, along with the output, have a look at the other modules (e.g. cadc or ipac.irsa) for examples. If you run the remote tests on this file, edit it until it passes. (`python setup.py test -t docs/desi/desi.rst -R`)

Reply:
I believe this issue is still holding tests back from passing